contracts/pool/ConcentratedLiquidityPool.sol

```diff
@@ -850,8 +850,12 @@ contract ConcentratedLiquidityPool is IConcentratedLiquidityPoolStruct, IPoolFac
             feeGrowthAbove1 = _feeGrowthGlobal1 - upper.feeGrowthOutside1;
         }
 
-        feeGrowthInside0 = _feeGrowthGlobal0 - feeGrowthBelow0 - feeGrowthAbove0;
-        feeGrowthInside1 = _feeGrowthGlobal1 - feeGrowthBelow1 - feeGrowthAbove1;
+        unchecked {
```

Comment:
#### update conditions

from `nearestTick` to `poolTick`

* reference : https://github.com/code-423n4/2021-09-sushitrident-2-findings/issues/13